templates/content/node--ucb-person.html.twig

```diff
@@ -18,7 +18,7 @@
 <article{{ attributes.addClass(classes) }} itemscope itemtype="https://schema.org/Person">
   <div class="container ucb-person-header">
     <h1 class="ucb-person-name">
-      {{ content.field_ucb_person_first_name }} {{ content.field_ucb_person_last_name }}
+      <span itemprop="givenName">{{ content.field_ucb_person_first_name.0 }}</span> <span itemprop="familyName">{{ content.field_ucb_person_last_name.0 }}</span>
```

Comment:
We lost the`|trim` on the first and last name content, that's probably still useful to correct simple user-entered data errors.  

We don't have good visibility or control over the Fastly optimization and given that this is already behaving differently with different browsers I want to have a real solution that we can rely on that's under out control.  Please change the whitespace character to an HTML non-breaking space.